dbms/src/Flash/Mpp/MPPTask.cpp

```diff
@@ -174,6 +175,8 @@ void MPPTask::registerTunnels(const mpp::DispatchTaskRequest & task_request)
     auto tunnel_set_local = std::make_shared<MPPTunnelSet>(log->identifier());
     std::chrono::seconds timeout(task_request.timeout());
     const auto & exchange_sender = dag_context->dag_request.rootExecutor().exchange_sender();
+    size_t tunnel_queue_memory_bound = getAverageThreshold(context->getSettingsRef().max_buffered_bytes_in_executor, exchange_sender.encoded_task_meta_size());
```

Comment:
Some worry that memory_buffer is not enough in large-scale clusters.
For a scale of 100 nodes, a default queue is only 1mb

Reply:
ok